x/evm/keeper/grpc_query.go

```diff
@@ -72,6 +71,50 @@ func (k Keeper) CosmosAccount(c context.Context, req *types.QueryCosmosAccountRe
 	return &res, nil
 }
 
+func (k Keeper) ValidatorAccount(c context.Context, req *types.QueryValidatorAccountRequest) (*types.QueryValidatorAccountResponse, error) {
```

Comment:
can you add a test for this query too? 🙏

Reply:
yes, added